src/components/pagination/PaginationSubBox.js

```diff
@@ -12,191 +12,250 @@ import CheckBlueImage from '../../images/icon-check-blue.svg'
+                    border-bottom: 0px solid #e6e6e6;
+                }
             }
-         }
-      }
-      @media screen and (max-width: 767px) {
-         border: 0px;
-         position: absolute;
-         min-height: 300px;
-         top: -80px;
-
-         .img {
-            top: 0;
-            right: 2px;
-         }
-         > .item {
+        }
+
+        .img {
+            width: 20px;
+            position: absolute;
+            top: 20px;
+            right: 20px;
             padding: 0px;
 
-            h2 {
-               font-size: 32px !important;
+            img {
+                cursor: pointer;
+            }
+        }
+        .text {
+            margin: 0 10% 0 0;
+            color: #24272a;
+            float: left;
+
+            .header {
+                font-family: 'benton-sans', sans-serif;
+            }
+            .content {
+                color: #999999;
+                padding-top: 12px;
+                line-height: 20px;
             }
-            > .list {
-               padding: 0;
+        }
+        
+        .send {
+            padding: 0 1.5em;
+            
+            button {
+                width: 100%;
+                margin-top: 1em;
+            }
+        }
+        
+        @media screen and (max-width: 991px) {
+            .img {
+                top: 6px;
+                right: 6px;
             }
             .title {
-               width: 100%;
-               padding-bottom: 12px;
-               
-               > .item {
-                  width: 100%;
-                  text-align: center;
-                  margin-left: 0;
-               }
-               .image {
-                  > div {
-                     margin: 0 auto;
-                  }
-               }
+                .item {
+                    margin-left: 0;
+                    text-align: center;
+                }
+            }
+            .image {
+                width: 100%;
+
+                > div {
+                    margin: 0 auto;
+                }
+            }
+        }
+        @media screen and (max-width: 767px) {
+            border: 0px;
+            position: absolute;
+            min-height: 300px;
+            top: -80px;
+
+            .img {
+                top: -12px;
+                right: 8px;
             }
-         }
-         .remove-connection {
-            padding-top: 0px;
-            border-top: 2px solid #e6e6e6;
-         }
-      }
-   }
+            .deauthorize-box {
+                .top {
+                    padding: 1em 0 1em 0;
+
+                    &.contacts {
+                        padding: 2em 0 0;
+                    }
+
+                    h2 {
+                        font-size: 32px !important;
+                    }
+                    .title {
+                        padding-bottom: 12px;
+                        .image {
+                            > div {
+                                margin: 0 auto;
```

Comment:
For future re-factor efforts: Let's utilize styled-components when appropriate to improve readability and make things more modular. Many levels of nesting makes it hard to follow.

Reply:
I know what you mean, it's problematic. But also I think that using styled component for small items is problematic to debug too because you end up with hashed classes in dev tools, which makes it harder to find in code.

I think the ideal way to use style components is to make as many reusable components as possible and define more things in GlobalStyles. Personally I don't like when css is defined inside react component because it's a bit messy, that's why in some cases I did put all css in one parent component (for example `SendMoneyContainer.js`), define css for child components there - but of course, it's not a good idea for some cases. 

At this moment css structure is a bit messy because at early stages the design for all pages was changing a lot, many new elements were added. But in the future, it would be best to rebuild the whole css structure, makes more reusable components and have more things in GlobalStyles. Maybe we can also export style components to separate files for every folder or something like that?

Long reply, but I think it's a big issue and we should discuss it in the future.

Reply:
Wonder if we should use this macro to make debugging style-components easier? https://stackoverflow.com/a/60887852/341267

I think I also prefer styles to be encapsulated in components, this seems to scale far better than all big previous projects I worked on which used global styles – they ultimately became very hard to manage mess and with no clear scope which styles impact which component.

So likely best not to use global styles except for basic stuff – like making default styles for HTML tags like `h1`, `h2`, links, etc

Reply:
> So likely best not to use global styles except for basic stuff

I agree, what I mean about "have more things in GlobalStyles" was to put in GlobalStyles most of CSS code that is repeated in files and that cannot be put (or it's unnecessary/overdone) to a reusable component. For example all font styles (like you mentioned), border styles, font colors (as it might be mixed for different font styles) etc.

> Not having to think about classNames is a major benefit IMO.

What I meant, showing on examples, is that having separate styled components for Header and Title here:
https://github.com/nearprotocol/near-wallet/blob/master/src/components/accounts/recovery_setup/RecoveryOption.js#L110
seems a bit unnecessary, but I start to think it might be a better approach - JSX structure looks much better for sure.

Reply:
> What I meant, showing on examples, is that having separate styled components for Header and Title here:

agree that it's less clear trade-off in this case. I think biggest advantage of having it as styled component is just that it's easier to extract it into separate reusable one when needed.